test/spec/modules/openxRtbBidAdapter_spec.js

```diff
@@ -778,17 +780,20 @@ describe('OpenxRtbAdapter', function () {
         expect(bid.currency).to.equal(bidResponse.cur);
       });
 
-      // TODO: Need to find how this is passed in
-      it('should return a transaction state', function () {
-        expect(bid.ts).to.equal(bidResponse.seatbid[0].bid[0].ext.ts);
-      });
-
       it('should return a brand ID', function () {
         expect(bid.meta.brandId).to.equal(bidResponse.seatbid[0].bid[0].ext.brand_id);
       });
 
       it('should return a dsp ID', function () {
-        expect(bid.meta.dspid).to.equal(bidResponse.seatbid[0].bid[0].ext.adv_id);
+        expect(bid.meta.networkId).to.equal(bidResponse.seatbid[0].bid[0].ext.dsp_id);
+      });
+
+      it('should return a buyer ID', function () {
+        expect(bid.meta.advertiserId).to.equal(bidResponse.seatbid[0].bid[0].ext.buyer_id);
+      });
+
+      it('should return adomain', function () {
+        expect(bid.meta.advertiserDomains).to.equal(bidResponse.seatbid[0].bid[0].adomain);
```

Comment:
this test is failing because its `adomain` in the adapter.  I looked through the code and there doesn't seem to be any big exchanges exposing this so we should obfuscate this until prebid 5.0 forces us to